Source/Core/DiscIO/Enums.cpp

```diff
@@ -159,11 +160,24 @@ Region CountryCodeToRegion(u8 country_code, Platform platform, Region expected_r
       return Region::NTSC_J;  // Korean GC games in English or Taiwanese Wii games
 
   case 'E':
-    if (expected_region == Region::NTSC_J)
-      return Region::NTSC_J;  // Korean GC games in English
-    else
+    if (platform != Platform::GameCubeDisc)
       return Region::NTSC_U;  // The most common country code for NTSC-U
 
+    if (revision)
+    {
+      if (*revision >= 0x30)
```

Comment:
It would be possible to put some kind of limit on it, but there's no specific number that's the right one. I'm not especially concerned about this, since the intention behind the feature is that it lets you check if you have a correct dump or not, and homebrew releases are not correct dumps, so I'm okay with showing errors for them even if the description isn't quite accurate.